CHARTER.md

```diff
@@ -13,6 +13,17 @@ The mission of Aggie Robotics is to provide practical educational opportunities
 
 ## Membership
 
+Membership in Aggie Robotics contains 3 levels: Journeyman Membership, Associate Membership, and Full Membership. All members will receive a `Member Badge`. Members can advance in level by `badging`. The `badging` process will be defined by the by-laws and published openly.
```

Comment:
Will badging process be quantitative, "Progress from level 1 -5. Now you are an Associate Member."? 

This could be a big selling point that gets the attention and retention of young students, who are into the video game scene. 

Perhaps within each subgroup, fun-name titles, and reward system associated with that title; can be considered.

i.e. I am a Journeyman, i have attended '3' meetings, so now i get a badge. For that badge i have earned, i get a nonofficial - respective name. Say, "Awesom -O". 
Badge 2:  "Roomba"
Badge 3: "KITT"
Badge 3: "Mega Man"
Badge 4: "Wall-E"
Badge 5: "Optimus Prime"
...
Badge 10: "The Terminator"

Thoughts?

Unofficial name titles can be anything, from famous computer scientists, or simply military ranks. 

Thoughts..?